Backend/core/db.py

```
@@ -1,119 +1,288 @@
-    for key in keys[1:]:
-        result[key] = float(r.execute_command('TS.GET', key)[1])
+    
+    if config.USE_TIMESERIES:
+        try:
+            data = r.execute_command('TS.GET', keys[0])
+            result['timestamp'] = data[0]
+            result[keys[0]] = float(data[1])
+            
+            for key in keys[1:]:
+                result[key] = float(r.execute_command('TS.GET', key)[1])
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)